<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>382</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="SceneView" name="sceneView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="applianceSidebar" native="true">
     <layout class="QVBoxLayout" name="applianceSidebarLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="topBar" native="true">
     <property name="styleSheet">
      <string notr="true">#topBar {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
stop:0 rgba(255, 255, 255, 255),
stop:0.190 rgba(232, 232, 232, 255),
stop:0.191 rgba(202, 202, 202, 255),
stop:1 rgba(237, 237, 237, 255))
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="applianceLeftBar" native="true">
        <layout class="QHBoxLayout" name="applianceLeftBarLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="contentSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="feedbackWidget" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>14</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PixmapButton" name="lbLike" native="true">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="pixmap" stdset="0">
            <pixmap resource="../3rdparty/likebackfrontend/likebackfrontend.qrc">:/likeback/like.png</pixmap>
           </property>
           <property name="fixedSize" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="fadeInactive" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapButton" name="lbFeature" native="true">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="pixmap" stdset="0">
            <pixmap resource="../3rdparty/likebackfrontend/likebackfrontend.qrc">:/likeback/feature.png</pixmap>
           </property>
           <property name="fixedSize" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="fadeInactive" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapButton" name="lbBug" native="true">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="pixmap" stdset="0">
            <pixmap resource="../3rdparty/likebackfrontend/likebackfrontend.qrc">:/likeback/bug.png</pixmap>
           </property>
           <property name="fixedSize" stdset="0">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="fadeInactive" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="GroupBoxWidget" name="modeWidget" native="true">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="16">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="16">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="16">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="16">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title" stdset="0">
         <string>SETUP</string>
        </property>
        <property name="checkable" stdset="0">
         <bool>true</bool>
        </property>
        <property name="checked" stdset="0">
         <bool>false</bool>
        </property>
        <property name="borderFlags" stdset="0">
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="transpBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Transparent Window</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="accelBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset resource="../fotowall.qrc">
             <normaloff>:/data/opengl-logo.png</normaloff>:/data/opengl-logo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="BlinkingToolButton" name="accelTestButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Test GL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="applianceRightBar" native="true">
        <layout class="QHBoxLayout" name="applianceRightBarLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="navBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#navBar {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #d8d8d8, stop:1 #eeeeee)
}
</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxWidget</class>
   <extends>QWidget</extends>
   <header>Shared/GroupBoxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SceneView</class>
   <extends>QGraphicsView</extends>
   <header>App/SceneView.h</header>
  </customwidget>
  <customwidget>
   <class>BlinkingToolButton</class>
   <extends>QToolButton</extends>
   <header>Shared/BlinkingToolButton.h</header>
  </customwidget>
  <customwidget>
   <class>PixmapButton</class>
   <extends>QWidget</extends>
   <header>Shared/PixmapButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sceneView</tabstop>
  <tabstop>transpBox</tabstop>
  <tabstop>accelBox</tabstop>
  <tabstop>accelTestButton</tabstop>
 </tabstops>
 <resources>
  <include location="../3rdparty/likebackfrontend/likebackfrontend.qrc"/>
  <include location="../fotowall.qrc"/>
 </resources>
 <connections/>
</ui>
